package/MDAnalysis/coordinates/XYZ.py

```diff
@@ -170,14 +172,16 @@ def __init__(self, filename, n_atoms=None, convert_units=True,
            parameter is no longer relevant and has been removed. If passing
            an empty universe, please use ``add_TopologyAttr`` to add in the
            required elements or names.
+        .. versionchanged:: 2.4.0
+           Chage open mode to be a keyword argument.
         """
         self.filename = filename
         self.remark = remark
         self.n_atoms = n_atoms
         self.convert_units = convert_units
 
         # can also be gz, bz2
-        self._xyz = util.anyopen(self.filename, 'wt')
+        self._xyz = util.anyopen(self.filename, mode)
```

Comment:
does opening in `'at'` automatically seek to the end of the file?  Either way, it's probably worth sanity checking the mode that was passed here.